examples/exempt_custom_fees/main.go

```diff
@@ -166,6 +142,27 @@ func main() {
 	 *
 	 * Collector 0.0.B sends 10_000 units of the token to 0.0.A.
 	 */
+
+	// First we transfer the amount from treasury account to second account
+	treasuryTokenTransderTransaction, err := hedera.NewTransferTransaction().
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** Typo: `treasuryTokenTransderTransaction` -> `treasuryTokenTransferTransaction`‏